src/utilities/buildASTSchema.js

```diff
@@ -112,6 +120,10 @@ export function buildASTSchema(
     throw new Error('Must provide a document ast.');
   }
 
+  if (!options || !(options.assumeValid || options.assumeValidSDL)) {
```

Comment:
@mjmahone We need separate `assumeValidSDL` check so developers are able to change set of validation rules(add or delete).
```js
const errors = validateSDL(ast, null, customSetOfRules);
if (errors.length !== 0) {
  // ...
}
const schema = buildASTSchema(ast, { assumeValidSDL: true });
```

Reply:
Ok this makes sense. A little bit annoying to add in that second key, but not the end of the world.

Reply:
Wait: are you thinking `assumeValid` is a superset of `assumeValidSDL` or disjoint? I would still ideally have it be a superset. We could add `assumeValidExecutableAST` or something that is disjoint with `assumeValidSDL`.

Reply:
@mjmahone My proposal:
`{}` => `validateSDL` + `validateSchema`
`{ assumeValidSDL: true }` => `validateSchema`
`{assumeValid: true }` => no checks